templates/emails/github.signup.hbs

```diff
@@ -8,13 +8,24 @@ Subject: Your Open Collective is on its way...
 <p>Hi!</p>
 {{/if}}
 
-<p> Thanks for signing up your GitHub repository: {{group.name}}. We are reviewing new requests as fast as we can. </p>
+<p> Thanks for signing up on Open Collective! Your collective is now live: <a href="{{config.host.website}}/{group.slug}">{{group.name}}</a>. While you wait for approval to start receiving donations, here's a quick guide to make the most out of it: </p>
 
-<p>While you wait, check out other <a href="https://opencollective.com/opensource">open source collectives</a> and see how they are spending the money they have raised.</p>
+<ol type="1">
+<li><strong>Make your collective your own.</strong> Login and hover over elements to edit them. </li>
+
+<li><strong>Add backers/sponsors to Github/Website.</strong> We'll submit a PR to your README to add backers & sponsors. Instructions to add your backers to a website <a href="https://github.com/OpenCollective/OpenCollective/wiki/Website-banner">here</a>.
```

Comment:
Setting up a collective is different. It's work and we expect people to put on the work.
Also I'm not adding any new point here, just modifying an existing one so that we are not the bottleneck to have them up and ready, especially if they are willing to put the work of doing the PR themselves.